ambition_utils/rrule/models.py

```diff
@@ -255,5 +252,45 @@ def save(self, *args, **kwargs):
         if self.rrule_params.get('until') and hasattr(self.rrule_params.get('until'), 'date'):
             self.rrule_params['until'] = self.rrule_params['until'].strftime('%Y-%m-%d %H:%M:%S')
 
+    def save(self, *args, **kwargs):
+        """
+        Saves the rrule model to the database. If this is a new object, the first next_scheduled time is
+        determined and set. The `dtstart` and `until` objects will be safely encoded as strings if they are
+        datetime objects.
+        """
+        self.pre_save_hooks()
+
         # Call the parent save method
         super().save(*args, **kwargs)
+
+    def get_next_dates(self, num_dates=20):
```

Comment:
Could this be a class method that just takes rrule params and sets up the modeling for you? So I could just call `RRule.generate_dates_from_rrule` or something?

Reply:
there could be a wrapper that calls this code so you can get the dates in one line instead of two.